config/jest.config.js

```diff
@@ -1,20 +1,22 @@
+require('dotenv').config() // TODO: Create env file in github actions
```

Comment:
Is this still a todo? If not, can we open an issue instead of adding comments in the code? 🙂